docs/deployment/landing_page.rst

```diff
@@ -15,6 +15,38 @@ your organization.
           implement minimum security requirements is sure to be noticed, and
           could undermine trust, discouraging possible sources.
 
+SecureDrop `maintains a directory of instances that meet our strict guidelines.
+<https://securedrop.org/directory/>`__ If you would like to be considered for
+inclusion in this directory, make sure your landing page is in compliance with
+the requirements below, then `send us a request using this form.
+<https://securedrop.org/directory/submit/>`__
+
+There are several benefits to being included in the SecureDrop directory. The
+most significant benefit is that it will be easier for potential sources to
+find your SecureDrop instance. Additionally, being included in the directory
+makes you eligible for a short onion address. This improves the experience by
```

Comment:
Tor has adopted the name "onion names" for this functionality, so I would suggest we standardize on that here as well. In case you have not seen it, we also have an [FAQ for onion names here](https://securedrop.org/faq/getting-onion-name-your-securedrop/), which we can cross-link here.